internal_worker_base.go

```diff
@@ -128,13 +128,17 @@ func (bw *baseWorker) Start() {
 	}
 
 	bw.isWorkerStarted = true
-	bw.logger.Info("Started Worker", zap.Int("RoutineCount", bw.options.routineCount))
+	traceLog(func() {
```

Comment:
Tried hiding these logs at worker_base, this is internal layer of worker. Added life cycle logs on the AggregatedWorker in internal_worker.go.  same for below comments as well.